cmd/okctl/main.go

```diff
@@ -168,8 +167,6 @@ func buildRootCommand() (*cobra.Command, *okctl.Okctl) {
 			o.Out = cmd.OutOrStdout()
 			o.Err = cmd.OutOrStderr()
 
-			o.SetFormat(core.EncodeResponseType(outputFormat))
```

Comment:
This doesn't have anything to do with client/server? I don't think we're using it though

Reply:
Format for http requests - that is going away :)